abnf/odata-aggregation-abnf.txt

```diff
@@ -194,8 +194,9 @@ traverseTrafo   = %s"traverse" OPEN
 
 recHierReference = BWS rootExpr BWS COMMA

```

Comment:
According to the spec, the first parameter must be "an expression of type Collection(Edm.EntityType)".
Since it is difficult to limit the syntax accordingly, I suggest to add a comment here to make implementors aware of this constraint.